### ISSUE TITLE
Time can be incorrectly scraped for parsing.

### ISSUE THREAD

**Issue by quetzaluz** [dotandbo/mxd-tracking#5]
Errors are thrown in `MxdTracker::TimeConverter.convert_time_to_utc` when the date is incorrectly scraped. Here's an example shipment in spree with this problem:

```
s = Spree::Shipment.find(1972834)
  Spree::Shipment Load (1.8ms)  SELECT  `spree_shipments`.* FROM `spree_shipments`  WHERE `spree_shipments`.`id` = 1972834 LIMIT 1
=> #<Spree::Shipment id: 1972834, tracking: "ZHN4AU", number: "P263370", cost: #<BigDecimal:a917008,'0.66E2',9(18)>, shipped_at: "2015-08-17 14:00:44", order_id: 4631972, address_id: nil, state: "shipped", created_at: "2015-08-17 13:21:30", updated_at: "2015-08-21 20:12:36", stock_location_id: 1, first_scanned_at: nil, delivered_at: nil, shipping_carrier_id: 98, shipping_carrier_method_id: 190, shipping_carrier_cost: #<BigDecimal:a916ce8,'0.22357E3',18(18)>, tracking_entered_at: "2015-08-17 14:00:44", promo_total: #<BigDecimal:a916bf8,'0.0',9(18)>, pre_tax_amount: #<BigDecimal:a916b58,'0.0',9(18)>, adjustment_total: #<BigDecimal:a916b08,'0.0',9(18)>, additional_tax_total: #<BigDecimal:a916950,'0.0',9(18)>, included_tax_total: #<BigDecimal:a9168d8,'0.0',9(18)>>
```

The raw scraped status data is this:
"Scheduled delivery date: Wed, September 02, 2015 - 12:00 PM to 02:00 PM"
Due to the calls to split and so forth, we only strip the following invalid timestamp from this result:
"2015 - 12:00 PM to 02:00 PM"

**Comment by salazareddie** [comment]
@quetzaluz will check it out when I get in

**Comment by quetzaluz** [comment]
This particular case was resolved by 049aa6632c123c1eb0f207b77da2cbcdb17bd542, closing.
